modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-impl/src/main/resources/META-INF/resources/render/form_handlers/metal_js.jspf

```diff
@@ -61,24 +61,37 @@
 		})
 	}
 
+	function getFormValues(form) {
+		return !form ? [] : form.getImmediateFields().map(
```

Comment:
@gianmarcobrunialti I know this is correct but is hard to read, a negational condiction in a Ternary operation, is a bit complex to read